src/appmixer/utils/csv/AddRow/AddRow.js

```diff
@@ -24,8 +24,9 @@ module.exports = {
             parseBooleans
         });
         let lock;
+        let lockExtendInterval;
         try {
-            lock = await context.lock(fileId, { retryDelay: 1000 });
+            lock = await context.lock(fileId, { ttl: 1000 * 60 * 1, maxRetryCount: 0 });
```

Comment:
is there a reason for the `maxRetryCount: 0`? Because that is a good feature. Imagine you have 5 messages going into this component. The first one gets the lock and starts modifying the csv. With retryCount 0, all the others get rejected and will be retried in a minute. With a retryDelay and retryCount, the 4 messages must not be directly rejected/put-to-retry-mechanism, but can wait here for a bit. When the first one releases the lock, the second will get it with and so on. Processing all those 5 messages can be faster (may not go to the `appmixer retry` mechanism, at all).